.github/script/call-canary-deploy-demo-prod

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+const { spawnOrFail, logger } = require('../../script/cli-utils');
+
+const myArgs = process.argv.splice(2);
+var demoCurrentVersion = myArgs[0];
+
+var npmLatestVersion = spawnOrFail('npm',['view', 'amazon-chime-sdk-js@latest', 'version']);
+var npmBetaVersion = spawnOrFail('npm', ['view', 'amazon-chime-sdk-js@beta', 'version']);
+
+// Removing line breaks from the strings before comparison
+npmLatestVersion = npmLatestVersion.replace(/(\r\n|\n|\r)/gm,'');
+npmBetaVersion = npmBetaVersion.replace(/(\r\n|\n|\r)/gm,'');
+demoCurrentVersion = demoCurrentVersion.replace(/(\r\n|\n|\r)/gm,'');
+
+logger.log('NPM latest version: ', npmLatestVersion);
+logger.log('NPM beta version: ', npmBetaVersion);
+logger.log('Demo current version: ', demoCurrentVersion);
+
+if (npmBetaVersion === npmLatestVersion || npmBetaVersion === '') {
```

Comment:
When will the `===` ever happen? Shouldn't you compare with major version part only?